### ISSUE TITLE
expose prop types

### ISSUE THREAD

**Issue by KutnerUri** [wbkd/react-flow#539]
Prop types are not accessible from the library.
It should be easy to expose them, at least with typescript 3.8+.

```tsx
export { ReactFlow as default } from './container/ReactFlow';
export type { ReactFlowProps } from './container/ReactFlow'; 
export { default as Handle } from './components/Handle';
export type { HandleProps } from './components/Handle';
```

etc

**Comment by moklick** [comment]
Hey. Do you mean the types or the React prop types?

**Comment by KutnerUri** [comment]
I mean the type of the props of the components.

I have a habit of extending components props, so future consumers could add arguments I did not expect.
This might explain it better: https://blog.bitsrc.io/the-vanilla-and-flavouring-pattern-fcf07942a57a

**Comment by chrtze** [comment]
Released in [6.1.1](https://github.com/wbkd/react-flow/releases/tag/6.1.1)
